lib/alchemy/engine.rb

```diff
@@ -52,6 +51,7 @@
 
 # Require hacks
 require_relative './kaminari/scoped_pagination_url_helper'
+require_relative './sass-support'
```

Comment:
Fair enough. This felt a little off to me too as I was doing it, so I will move this line 47.